### ISSUE TITLE
Icons changed don't show correctly

### ISSUE THREAD

**Issue by dariocdj** [souliss/soulissapp#116]
I've 24 nodes and If I change nodes icons in main list app show me wrong one in most nodes

**Comment by long-n** [comment]
I have same problem.
I am newbie to Souliss and i have 2 nodes. When i run soulissapp on my Samsung Galaxy Note 10.1 Android ver. 4.4.2, i can change icons for both 2 nodes and they show correctly. But on my newer Samsung Galaxy Tab S2 Android ver. 5.1.1, they show the wrong one. For example, i change the icon for the gateway node, the icon i picked applied to the peer node.

**Comment by shineangelic** [comment]
is this confirmed in 2.3.2 ? thanks

**Comment by shineangelic** [comment]
@dariocdj can you replicate this recently? Otherwise to me its closed

**Comment by dariocdj** [comment]
No, you can close this issue, also if i can give a suggest, icon set is too large for souliss, and thereis lots of useless icons. You must search lots of time a lamp for example.
